packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -93,6 +93,10 @@
 		box-shadow: 0 $border-width 0 0 rgba($color: #000, $alpha: 0.133); // 0.133 = $gray-200 but with alpha.
 	}
 
+	&.is-contentonly-mode {
+		background-color: $gray-900;
+	}
```

Comment:
Is this change related, I'm guessing it's a lingering change right?

Reply:
yes i have updated the code.